vhd/packer/install-dependencies.sh

```diff
@@ -304,7 +304,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
-1.2.1
+v1.2.2_hotfix
```

Comment:
we need to get rid of the `v` here (it's automatically inserted in the string concatenation below in L310)

Reply:
Fixed it.